src/components/Footer.js

```diff
@@ -17,10 +18,11 @@ const links = [
 ];
 
 const Footer = () => {
-  const classes = useMyStyles();
+  const classesCommon = common();
```

Comment:
if you want to use them like this, you should extract them outside component, on every rerender they will be created again
but, if you extract them outside component, why just not import them as ready to use:
`import commonStyles from "../styles/common"` 
and then use like 
```className={`${commonStyles.gray} ${footerStyles.footer}`}"```

Reply:
Can't extract them outside component because makeStyles returns a hook - this way is has to be called inside. I might change styling method when Material UI v5 comes out.